theano/tensor/nnet/ctc.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+import theano
+import theano.tensor as T
+from theano import gof
+from theano.gof import local_optimizer
+from theano.tensor.opt import register_canonicalize
+from theano.tensor.opt import register_stabilize
+from theano.gradient import grad_undefined
+
+import os
+
+class ConnectionistTemporalClassification(gof.COp):
+    __props__ = ()
```

Comment:
`"computeGradient"` should be a prop.

Also, to keep syntax coherent, I suggest to use `compute_gradient` instead of `computeGradient` for the name of that class member !